plugins/veeValidate.ts

```diff
@@ -0,0 +1,21 @@
+import { Plugin } from '@nuxt/types/app';
+import { configure, extend } from 'vee-validate';
+import { required } from 'vee-validate/dist/rules';
+
+// Rules
+extend('required', required);
+
+extend('mustBeUrl', (value: string): boolean => {
+  return /^https?:\/\/.+/.test(value);
+});
+
+const veeValidate: Plugin = ({ app }) => {
+  configure({
+    // FIXME: Ts doesn't like the returned value to defaultMessage
+    defaultMessage: (_field, values) => {
+      // values._field_ = app.i18n.t(`fields.${field}`);
+      return app.i18n.t(`validation.${values._rule_}`, values);
```

Comment:
```suggestion
      return app.i18n.t(`validation.${values._rule_}`, values).toString();
```
This should fix the TS errors